### ISSUE TITLE
Bug: Can't connect to db

### ISSUE THREAD

**Issue by awebeer256** [gutmensch/docker-dmarc-report#39]
This may or may not be a duplicate of #26.

I'm using `podman-compose`. My compose file is the same as in the readme, plus @marvinscham's additions in #38, plus changing the outward-facing port to 3000 so I can run it rootless (I only need to access it from localhost).

<details><summary>docker-compose.yml</summary>

```
version: "3.6"

services:
  dmarc-report:
    image: "gutmensch/dmarc-report:latest"
    hostname: dmarc-report
    container_name: dmarc-report
    depends_on:
      - db
    ports:
      - "3000:80"
    environment:
      - "REPORT_DB_HOST=db"
      - "REPORT_DB_PORT=3306"
      - "REPORT_DB_NAME=dmarc_report"
      - "REPORT_DB_USER=dmarc_report"
      - "REPORT_DB_PASS=dbpassword"
      - "PARSER_IMAP_SERVER=mail"
      - "PARSER_IMAP_PORT=143"
      - "PARSER_IMAP_USER=<redacted>"
      - "PARSER_IMAP_PASS=<redacted>"
      - "PARSER_IMAP_READ_FOLDER=Inbox"
      - "PARSER_IMAP_MOVE_FOLDER=processed"
      - "PARSER_IMAP_MOVE_FOLDER_ERR=error"

  db:
    image: mariadb:10
    command: --skip-innodb-read-only-compressed
    environment:
      - "MYSQL_ROOT_PASSWORD=dbrootpassword"
      - "MYSQL_DATABASE=dmarc_report"
      - "MYSQL_USER=dmarc_report"
      - "MYSQL_PASSWORD=dbpassword"
    volumes:
      - ./dmarc-report-db:/var/lib/mysql
    healthcheck:
      test: ["CMD", "mysqladmin", "ping", "-h", "localhost", "-uroot", "-pdbrootpassword"]
      interval: 10s
      timeout: 10s
      retries: 5

```
</details>

When I run `podman-compose up -d`, the `dmarc-report` container immediately crashes.

`> podman logs dmarc-report`

```
CRIT: Dmarc reports could not be parsed. Check your IMAP and MYSQL Settings.
DEBUG: Parsing failed with the following output:

DBI connect('database=dmarc_report;host=db;port=3306','dmarc_report',...) failed: Can't connect to MySQL server on 'db' (111) at /usr/bin/dmarcts-report-parser.pl line 273.
dmarcts-report-parser.pl: Cannot connect to database
```

**Comment by marvinscham** [comment]
If `podman-compose` supports this, you could try separately starting the database first and waiting a minute to make sure it's fully started before starting the `dmarc-reports` container. The command should be something like `podman-compose start dmarc-db-1`.

**Comment by gutmensch** [comment]
@awebeer256 This image cannot make sure you are running your database correctly or that you schedule the service correctly after it's available - the readme shows an example of how it could possibly look, but it's not a bug of the repo content itself. Sorry, but helping with your podman setup is out of scope for me, but please feel free to share and/or PR other working examples if you have figured something out. :)
